script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -0,0 +1,52 @@
+countries = %w(
+  belgium
+  china
+  ecuador
+  egypt
+  finland
+  lebanon
+  macao
+  mongolia
+  morocco
+  norway
+  peru
+  philippines
+  qatar
+  south-korea
+  thailand
+  turkey
+  united-arab-emirates
+  vietnam
+)
+
+payment_method_partial = {
+  'default'     => 'pay_by_cash_or_credit_card_no_cheque',
```

Comment:
What's the purpose of this default value? It doesn't look as if it's being used in the app code.